tools/gencode.ts

```diff
@@ -117,13 +117,15 @@ const extractHandlersMetadata: () => Promise<
   GencodeIpcHandler[]
 > = async () => {
   // Load list of file paths in handlers directory, recursively visiting subfolders
-  const fileNames = (await recursivelyReadDirectory(HANDLERS_DIRECTORY)).map(
-    (filePath: string) => filePath.split(`${HANDLERS_DIRECTORY}/`)[1]
+  const filePaths = (await recursivelyReadDirectory(HANDLERS_DIRECTORY)).map(
+    (filePath: string) => {
+      return path.resolve(filePath);
```

Comment:
nitpick but can inline this `.map((filePath: string) => path.resolve(filePath))` or even (assuming arguments align) point-free notation style `.map(path.resolve)`

Reply:
Weirdly arguments don't align but I have reduced the number of lines. Thanks for pointing it out